app/main/lib/image_similarity.py

```diff
@@ -60,6 +81,23 @@ def add_image(save_params):
     db.session.rollback()
     raise e
 
+def add_image_pdq(save_params):
+  try:
+    if save_params.get("doc_id"):
+      print("ha")
+      delete_image(save_params)
+    image = ImageModel.from_url(save_params['url'], save_params.get('doc_id'), save_params['context'], save_params.get("created_at"))
```

Comment:
don't create a new ImageModel object, populate from the DB and only calculate and update pdq